src/core/dbus-unit.c

```diff
@@ -2437,10 +2437,6 @@ int bus_unit_set_properties(
                 if (r < 0)
                         return r;
 
-                if (!UNIT_VTABLE(u)->bus_set_property)
```

Comment:
Hm? Wait, `bus_set_property` is not set for .target and .device??

Reply:
Yeah, because they don't have any type-specific properties. So it should be OK with the updated version.